packages/core/test/financial-templates/expiring-multiparty/Liquidatable.js

```diff
@@ -604,13 +604,13 @@ contract("Liquidatable", function(accounts) {
       // After the liquidation the liveness timer on the withdrawl request should be re-set to the current time +
       // the liquidation liveness. This opens the position up to having a subsequent liquidation, if need be.
       const liquidation1Time = await liquidationContract.getCurrentTime();
-      assert(
```

Comment:
This is the second part of the false positive. No matter what you throw into this assert, it returns true.